src/http-client.js

```diff
@@ -34,10 +34,27 @@ export class HttpClient {
     this.requestProcessorFactories.set(HttpRequestMessage, createHttpRequestMessageProcessor);
     this.requestProcessorFactories.set(JSONPRequestMessage, createJSONPRequestMessageProcessor);
     this.pendingRequests = [];
+    this.interceptors = [];
     this.isRequesting = false;
   }
 
   /**
+   * Add new interceptor
+   *
+   * NOTE: Interceptors are stored in reverse order. Inner interceptors before outer interceptors.
```

Comment:
This is great info, but it's an implementation detail that shouldn't appear in documentation. I'd move it into a normal code comment inside the method.